feat(curriculum): add steps for report card printer workshop (#64801)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-report-card-printer/69461e4f98594012939bc1c6.md

---
id: 69461e4f98594012939bc1c6
title: Step 1
challengeType: 0
dashedName: step-1
---

# --description--

In this workshop, you will learn about Python data types by building a simple report card printer.

Start by declaring a variable called `name` and assign it the value `'Alice'` like the example below.

```py
greeting = 'Hello World'

> ```

Author: i see. i'll move the example up for the rest as well!

---
id: 69461e4f98594012939bc1c6
title: Step 1
challengeType: 0
dashedName: step-1
---

# --description--

> In this workshop, you will learn about Python data types by building a simple report card printer.

Reviewer: ```suggestion
In this workshop, you will practice Python data types by building a simple report card printer.
```

As you learned in previous lessons, variables are assigned in this way:

```py
greeting = 'Hello World'
```

> 

Reviewer: ```suggestion

In the example above, the variable `greeting` has the value of a string, which is a sequence or characters surrounded by quotes.

```

Create a variable called `name` and assign it the string `Alice`. Remember to surround the value with either single or double quotes as in the example above.

# --hints--

You should have a `name` variable.

```js

> assert(has_variable("name"));

Reviewer 2: ```suggestion
({ test: () => runPython("assert _Node(_code).has_variable('name')") })
```

I am doing this one as an example

> assert(has_variable("name"));

Author: Thank you for reviewing this on Christmas. I was honestly kind of confused when I wrote them and just tried to finish them first. I'll get into fixing them soon. Happy holidays!
